src/main/java/core/basesyntax/service/calculationservice/ServiceFruits.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service.calculationservice;
+
+public interface ServiceFruits {
+    String calculationFruits(String nameFruit);
+}
```

Comment:
```suggestion
public interface FruitService {
    void processTransactions (List<FruitTransaction> transactions);
}
```

This service should process all parsed transactions. It should use strategy to get a specific handler and use this handler to process transaction